pkg/backporter/backporter_test.go

```diff
@@ -20,7 +20,7 @@ var (
 	landingPage = fmt.Sprintf(htmlPageStart, "Home") + helpTemplateConstructor + htmlPageEnd
 )
 
-var allTargetVersions = sets.NewString("4.0.0", "4.1.0", "4.4.z")
+var allTargetVersions = sets.NewString("3.11.z", "4.0.z", "4.1.z", "4.2.z", "4.3.z", "4.4.z", "4.5.z", "4.6.0")
```

Comment:
Why is this a global?

Reply:
I thought global would be appropriate since its used in multiple tests and if it were to change at some point it should ideally be changed for all the tests. Do you think its appropriate to create separate copies of it for each test?

Reply:
Yeah and the tests don't necesarily need it filled with all available versions. They only need something that is relevant for what they are testing